vaadin-spring-boot-starter/pom.xml

```diff
@@ -27,16 +27,6 @@
             <id>vaadin-prereleases</id>
             <url>https://maven.vaadin.com/vaadin-prereleases</url>
         </repository>
-        <repository>
```

Comment:
Should the https://maven.vaadin.com/vaadin-prereleases have:

<snapshots>
                <enabled>true</enabled>
</snapshots>
?

Reply:
by default, snapshots are set to true.
but yeah.. probably have the snapshot setting is more clear. will add it.